triggers/daml/Daml/Trigger.daml

```diff
@@ -89,7 +89,7 @@ data Trigger s = Trigger
   -- ^ Update the user-defined state based on a transaction or
   -- completion message.  It can manipulate the state with `get`, `put`,
   -- and `modify`, or query the ACS with `query`.
-  , rule : Party -> s -> TriggerA ()
+  , rule : Party -> TriggerA s ()
```

Comment:
We should note that `initialize` is once again the odd one out, as there is nothing to `get` at first.